src/login/LoginMain/LoginMain.scss

```diff
@@ -0,0 +1 @@
+@import "../../variables.scss";
```

Comment:
as sass is all set up, maybe we should make an index.scss file in start importing stylesheets into it? Then we can control the order in which they bundle and perhaps be more clever with the cascading.